{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-14614",
      "ASSIGNER": "cve@mitre.org"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-476"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://bugzilla.kernel.org/show_bug.cgi?id=200419",
          "name": "https://bugzilla.kernel.org/show_bug.cgi?id=200419",
          "refsource": "MISC",
          "tags": [
            "Issue Tracking",
            "Technical Description"
          ]
        },
        {
          "url": "http://www.securityfocus.com/bid/104917",
          "name": "104917",
          "refsource": "BID",
          "tags": [
            "Third Party Advisory",
            "VDB Entry"
          ]
        },
        {
          "url": "https://lists.debian.org/debian-lts-announce/2019/03/msg00017.html",
          "name": "[debian-lts-announce] 20190315 [SECURITY] [DLA 1715-1] linux-4.9 security update",
          "refsource": "MLIST",
          "tags": []
        },
        {
          "url": "https://usn.ubuntu.com/3932-2/",
          "name": "USN-3932-2",
          "refsource": "UBUNTU",
          "tags": []
        },
        {
          "url": "https://usn.ubuntu.com/3932-1/",
          "name": "USN-3932-1",
          "refsource": "UBUNTU",
          "tags": []
        },
        {
          "url": "https://usn.ubuntu.com/4094-1/",
          "name": "USN-4094-1",
          "refsource": "UBUNTU",
          "tags": []
        },
        {
          "url": "https://usn.ubuntu.com/4118-1/",
          "name": "USN-4118-1",
          "refsource": "UBUNTU",
          "tags": []
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An issue was discovered in the Linux kernel through 4.17.10. There is an out-of-bounds access in __remove_dirty_segment() in fs/f2fs/segment.c when mounting an f2fs image."
        }
      ]
    }
  },
  "configurations": {
    "nodes": [
      {
        "operator": "OR",
        "cpe_match": [
          {
            "vulnerable": true,
            "cpe23Uri": "cpe:2.3:o:linux:linux_kernel:*:*:*:*:*:*:*:*"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "vectorString": "AV:N/AC:M/Au:N/C:N/I:N/A:C",
        "accessVector": "NETWORK",
        "accessComplexity": "MEDIUM",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "COMPLETE",
        "baseScore": 7.1
      },
      "severity": "HIGH",
      "exploitabilityScore": 8.6,
      "impactScore": 6.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": true
    }
  },
  "publishedDate": "2018-07-27T04:29Z",
  "lastModifiedDate": "2019-04-03T01:29Z"
}